manager/testdata/new-blueprints/copy-read-blueprint.yaml

```diff
@@ -0,0 +1,96 @@
+apiVersion: app.m4d.ibm.com/v1alpha2
+kind: Blueprint
+metadata:
+  name: copy-and-read
+  labels:
+spec:
+  cluster: theshire
+  appSelector: "app:demoapp"
+  appCluster: theshire
+  flow:
+  - name: copy-step
+    assetID: 'm4d-notebook-sample/paysim-csv'
+    templateSpec:
+      chart: ghcr.io/mesh-for-data/m4d-implicit-copy-batch:0.1.0
+    moduleLabel: "mod: demoapp-copy-step"
+    frequency: "0 1 * * *"
+    source:
+      vault:
+        address: http://vault.m4d-system:8200
+        authPath: /v1/auth/kubernetes/login
+        role: module
+        secretPath: /v1/kubernetes-secrets/paysim-csv?namespace=m4d-notebook-sample
+      format: parquet
+      connection:
+        s3:
+          endpoint: localhost:8001
+          bucket: srcbucket
+          object: data.parq
+    destination:
+      vault:
+        address: http://vault.m4d-system:8200
+        authPath: /v1/auth/kubernetes/login
+        role: module
+        secretPath: /v1/kubernetes-secrets/paysim-csv?namespace=m4d-notebook-sample
+      format: parquet
+      connection:
+        s3:
+          endpoint: localhost:8001
+          bucket: destbucket
+          object: data.parq
+    transformations:
```

Comment:
in m4dmodule it is called actions - lets have consistent term in the new blueprints.

Reply:
I'll rename it in the m4dmodule. It's called transformations in the configuration design document as well. Please tell me if you think it should be called actions anyways. Then we should rename it everywhere.

Reply:
+1 for actions everywhere